### PR TITLE
Update webappService.sh

### PR DESCRIPTION
**Pull request by Snile826**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [ ] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/CHANGES.md).

### CODE REVIEW

**Comment by kezhenxu94** [comment]
@Snile826 please don't submit so many pull requests, you can do it in a single pull request, just push to the same branch

**Comment by kezhenxu94** [comment]
@Snile826 use the same branch to edit the files if you insist using GitHub Web UI to submit pull request https://github.com/Snile826/skywalking/tree/patch-3

File: dist-material/bin/webappService.sh [hunk 1]
```diff
@@ -14,14 +15,12 @@
...
 
-#!/usr/bin/env sh
-
 PRG="$0"
 PRGDIR=`dirname "$PRG"`
 [ -z "$WEBAPP_HOME" ] && WEBAPP_HOME=`cd "$PRGDIR/.." >/dev/null; pwd`
 
 WEBAPP_LOG_DIR="${WEBAPP_LOG_DIR:-${WEBAPP_HOME}/logs}"
-JAVA_OPTS=" -Xms256M -Xmx512M"
+JAVA_OPTS="${JAVA_OPTS:- -Xms256M -Xmx512M}"
```
Comment (wu-sheng) [inline dist-material/bin/webappService.sh:23]: What is this change?

**Comment by wu-sheng** [comment]
I am not sure about the purpose of these PRs, and edit on the GitHub and open pull request, makes me concerned about these changes are even not tested.

**Pull request closed without merging**